src/show/components/Split/index.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import React, { Fragment } from "react";
+import LeftImage  from "../../assets/img/image.jpg";
+
+const Split = ({ leftContent = Fragment, rightContent = Fragment }) => {
+  return (
+    <div className="h-screen flex">
+      <div className="w-full h-full">{leftContent}
+      <img src={LeftImage} alt="Left side" className="object-cover" />
```

Comment:
use Split on the SIgnIn and do something like this
 ```
<Split leftContent = {<img src={LeftImage} alt="Left side" className="object-cover" />}
rightContent={ <div className="flex justify-center mt-6">
      <div className="w-64 bg-white">
        <h2 className=" text-2xl font-bold mb-4">Sign in</h2>
        <form onSubmit={handleSubmit} className="signin-form">
          <TextField
            label="Email"
            type="email"
            name="email"
            placeholder="Enter Email"
            value={inputs.email}
            onChange={(value) => handleInputChange('email', value)}
            required
          />
          <TextField
            label="Password"
            type="password"
            name="password"
            placeholder="Enter Password"
            value={inputs.password}
            onChange={(value) => handleInputChange('password', value)}
            required
          />
          <Button
            type="submit"
            isFormValid={isFormValid}
          >
            Sign in
          </Button>
          <p className="mt-1 text-center font-semibold text-xs text-pink">Forgot Password?</p>
        </form>
      </div>
    </div>}
/>
```
this way we can pass different images here for different screen . remove the img from here

Reply:
done